apps/services/party-letter-registry-api/README.md

```diff
@@ -0,0 +1,8 @@
+# Party Letter Registry
+
+## About
+
+**This service needs to be accessed by users through another service providing appropriate authentication for getting this data**  
+The consuming application should validate that the requesting user is in managers  
+This service contains the political party letter registry.  
+The source of the data contained in this service is the Ministry of Justice and the party letter application currently contained within the application system.
```

Comment:
How do you plan to prefill the database with existing party letters?

Reply:
We plan to implement a import script using the existing seeder functionality in sequelize.
The database would then be prefilled with data from a xlsx file, allowing us to keep that out of the public repository.
This will also serve as the method of cose to update data in the database until we implement an admin panel.

The plan is to implement a admin panel to manage these letters and make it accessible to the Ministry of justice to enable them to maintain this data.
We decided to postpone the admin panel until the agency system (umboðskerfi) has been implemented.
The agency system should also allow us to omit the `managers` field from this database.